crates/storage/codecs/derive/src/compact/generator.rs

```diff
@@ -232,3 +236,26 @@ fn generate_to_compact(fields: &FieldList, ident: &Ident, is_zstd: bool) -> Vec<
 
     lines
 }
+
+/// Function to extract the crate path from #[reth_codecs(crate = "...")] attribute.
+
+fn parse_reth_codecs_path(attrs: &[Attribute]) -> TokenStream2 {
```

Comment:
if there's no `crate` attribute then this should fallback to  just `reth_codecs`